pandas/io/parsers.py

```diff
@@ -101,7 +100,8 @@ class ParserWarning(Warning):
 keep_default_na : bool, default True
     If na_values are specified and keep_default_na is False the default NaN
     values are overridden, otherwise they're appended to
-parse_dates : boolean, list of ints or names, list of lists, or dict, default False
+parse_dates : boolean, list of ints or names, list of lists, or dict,
+    default False
```

Comment:
This begs a general question: what to do with parameters that accept a ton of different types of input? I don't think we are doing users any favors by making a long line listing them all out; it would be better to have a bulleted list below the parameter name if that will render properly in the docs. Thoughts?